general/session.md

```diff
@@ -0,0 +1,26 @@
+# Session API
+
+## Workflow
+
+### Getting
+
+When getting a value stored in Session, we are deserializing it by removing the `serialize://` string from the data we retrieve. After the data is deserialized we get access to the data requested, stripped of any serialization.
```

Comment:
This seems incorrect. Maybe something like this instead?

---

`getSessionValue(key)` retrieves string data from the user's session by making a `fetch` request to the server at [a `portal/session_click` endpoint](https://github.com/liferay/liferay-portal/blob/2d2e8ae74e9e7b33a07328277da64f54db576a04/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/session.es.js#L34-L36). If the retrieved data begins with the `serialize://` prefix, it is stripped off and the remaining text is parsed as JSON using `JSON.parse()`. Otherwise, the data string is returned as-is.